website/docs/Use-Cases/Task-Oriented-AutoML.md

```diff
@@ -1,4 +1,4 @@
-# Task Oriented AutoML
+# GenericTask Oriented AutoML
```

Comment:
Is this and other changes from Task->GenericTask in the documentation a mistake?

Reply:
Hey! Thanks for the review.  Yeah, this and a couple of other instances appear to be a mistaken replacement. Likely Pycharm refactor going over-the-top. I've fixed them up now